CHANGELOG.md

```diff
@@ -4,8 +4,7 @@ x.x.x Release notes (yyyy-MM-dd)
 * None.
 
 ### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
-* None.
+* Fixed and issue with `toJSON` where data from a different objects could be serialized. ([#3254](https://github.com/realm/realm-js/issues/3254))
```

Comment:
"Fixed and issue" --> "Fixed an issue"
"a different objects" --> "a different object"

Do we know which version the detect was introduced?